datasets/_global/shu_uyghur_companies/crawler.py

```diff
@@ -0,0 +1,42 @@
+import openpyxl
+
+from zavod import Context, helpers as h
+
+
+def crawl(context: Context):
+    doc = context.fetch_html(context.data_url, cache_days=1)
+    url = doc.xpath(
+        './/a[contains(text(), "Companies Operating in the Uyghur Region")]/@href'
+    )
+    path = context.fetch_resource("source.xlsx", url[0])
+    workbook: openpyxl.Workbook = openpyxl.load_workbook(path, read_only=True)
+    for row in h.parse_xlsx_sheet(
+        context, sheet=workbook["1. Companies Operating in XUAR"], skiprows=1
```

Comment:
I think it's a good idea to get into a habit of asserting that new worksheets aren't introduced that we would have wanted to scrape. One way of doing that is asserting that set(workbook.sheets) = {'1. Companies Operating in XUAR', '2. Export Relevant Categories'}